_includes/featuredbox.html

```diff
@@ -8,9 +8,9 @@
                     <div class="wrapthumbnail">
                         <a href="{{ site.baseurl }}{{ post.url }}">
                             {% if site.lazyimages == "enabled" %}
-                            <img class="featured-box-img-cover lazyimg"  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image contains "://" %}{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image }}{% endif %}">
+                            <img class="featured-box-img-cover lazyimg"  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image-location contains "://" %}{{ site.image-location}}thumbnail_{{post.image }}{% else %}{{ site.baseurl }}/{{ site.image-location }}thumbnail_{{ post.image }}{% endif %}">
```

Comment:
Is `post.image-location` correct here? It is different in other _includes files.